demo/pkg/injector/context.go

```diff
@@ -0,0 +1,33 @@
+package injector
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/99designs/gqlgen/graphql/handler/transport"
+)
+
+type contextKey string
+
+const (
```

Comment:
Could you explain a bit why we need to store the payload and header?

Reply:
We need it because gqlgen doesn't expose the request header nor the WS initial payload to resolver functions. I'll add a doc.go file explaining that.

Reply:
Oh it is part of the demo. I didn't see it.